packages/app/src/components/Root/Root.tsx

```diff
@@ -81,6 +90,20 @@ const SidebarLogo = () => {
   );
 };
 
+const ProfileIcon = () => {
+  const { avatar, settings } = useSidebarLogoStyles();
+  const { profile: { picture }, displayName } = useUserProfile();
+  return (
+    <>
+      {picture ? <Avatar
+        src={picture}
+        alt={displayName}
+        className={avatar}
+      /> : <SettingsIcon className={settings} />}
+    </>
+  )
```

Comment:
```suggestion
if (picture) {
  return <Avatar src={picture} alt={displayName} className={avatar} />;
}
return <SettingsIcon className={settings} />;
```

Just a styling suggestion about the conditional. Considering we're only deciding between showing two components, we could just return one or the other like above.

I'm fine with how it is though